docs/install.md

```diff
@@ -7,6 +7,16 @@ documentation on the Sandstorm install script.](administering/install-script.md)
 
 Sandstorm requires Linux x86_64, with kernel version 3.13 or later.
 
+This page documents a number of ways to install Sandstorm, specifically:
+
+- [Most common: Downloading an executing our carefully-written install script, verified by HTTPS](#option-1-https-verified-install)
+- [Downloading the shell script via GitHub](#option-2-github-verified-install)
+- [Using PGP to verify the authenticity of the script](#option-3-pgp-verified-install)
+- [Installing Sandstorm from source code](#option-4-installing-from-source)
+- [Using Ansible, Puppet, or other configuration management tools](#option-5-integrating-with-configuration-management-systems-like-ansiblepuppet)
+- [Using Docker to run Sandstorm, with Sandstorm managing automatic updates](#option-6-using-sandstorm-within-docker)
+- [Running Sandstorm on Mac/Windows systems via Vagrant & VirtualBox](#option-7-use-vagrant-when-deploying-on-mac-or-windows)
```

Comment:
Arguably this section should mention vagrant-spk; I may add that later. Hopefully not a show-stopper and hopefully not something that needs pre-commit review, though docs review is always welcome.